data_processing_pipeline/6_visualize/src/build_svg.R

```diff
@@ -46,7 +46,8 @@ prepare_svg_data <- function(raw_dat, start_yr, end_yr) {
     ungroup() %>% # just in case it's grouped (causes weird issues)
     # Remove potential missing info
     filter(!is.na(year), !is.na(state), !is.na(n_gages)) %>% 
-    filter(year %in% start_yr:end_yr)
+    filter(year %in% start_yr:end_yr) %>% 
+    filter(state != "VI") # We aren't using the Virgin Islands
```

Comment:
Instead of filtering out a single state/terr in code, could you instead use the `cartogram_loc_config` to define which states are included or not? That way if you update this to include VI, you are only updating the cart locations and not changing any data prep code? 

This is a minor thing, and not a very likely hypothetical. But suggesting it because it is likely better to have a single decision point or filtering location for this kind of thing.